encoder.py

```diff
@@ -108,6 +108,13 @@ def _get_pooling(self, feature_map: 'np.ndarray') -> 'np.ndarray':
 
     @requests
     def encode(self, docs: Optional[DocumentArray], parameters: Optional[Dict] = None, **kwargs):
```

Comment:
parameters is not Optional, it will always exist

Reply:
thanks!